packages/remix-node/index.ts

```diff
@@ -5,10 +5,10 @@ sourceMapSupport.install();
 export { AbortController } from "abort-controller";
 
 export type {
-  HeadersInit,
-  RequestInfo,
+  // HeadersInit,
+  // RequestInfo,
   RequestInit,
-  ResponseInit,
+  // ResponseInit,
```

Comment:
Let's try and keep these exports if we can, even though they are really just re-exports of global DOM types, so we don't have a breaking change here.